### ISSUE TITLE
Build a multi-container Dapr application can't run successfully

### ISSUE THREAD

**Issue by XiaoGuiCC** [dotnet/docs#29533]
The 《Build a multi-container Dapr application》 section seems missing implementation code for method: 'weatherforecast'  of 'MyBackEnd', the demo can't run successfully.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 83e95a3f-447b-8ca0-beb5-f0363896fe29
* Version Independent ID: b24e3ee5-404c-98a2-042e-6ac2d363b842
* Content: [Get started with Dapr](https://docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/getting-started)
* Content Source: [docs/architecture/dapr-for-net-developers/getting-started.md](https://github.com/dotnet/docs/blob/main/docs/architecture/dapr-for-net-developers/getting-started.md)
* Product: **dotnet-architecture**
* GitHub Login: @amolenk
* Microsoft Alias: **nanil**

**Comment by XiaoGuiCC** [comment]
Yep, I created a wrong type project. But it couldn't run still after I changed the MyBackEnd project to ASP.NET Core Web API.
After I press F5, nothing happen. The program look compiler successfully. But no web page appear.
![Uploading BE2090F5-385D-4526-BF29-FB203EB8F886.jpeg…]()
If press the "OPEN IN BROWER" of myfrontend:dev in docker, it will show a error page.
Is it related to my os( macos with m1)?

**Comment by amolenk** [comment]
Ah yes, the tutorial was written for Visual Studio 2022 on Windows. I'll try the tutorial myself on my M1 Mac Mini and see if I can get it to work.

**Comment by robinhultman** [comment]
I managed to get it running on MacOS with Rider. The example doesn't bind to any port so I guess that Visual Studio does some magic to set the port binding.

You could update the docker compose file to set up the port binding for the frontend

`  myfrontend:
    image: ${DOCKER_REGISTRY-}myfrontend
    build:
      context: .
      dockerfile: MyFrontEnd/Dockerfile
    ports:
      - "8080:80"`

**Comment by erjain** [comment]
Thank you so much, folks. I have tested the tutorial and it is working fine now. The issue can be closed.
